cdap-gateway/src/test/java/co/cask/cdap/gateway/router/ServerResource.java

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
this was moved out from previous NettyRouterPipelineTest, plus added two new endpoints (echo and repeat) to the HTTP handler.